tdvt/samples/connector-tests/serverversion.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<testCases>
+	<testCase>
+		<connection class='postgres_odbc' server='postgres.test.tsi.lan' dbname='TestV1' username='test' password='password' tablename='Calcs'/>
```

Comment:
To be consistent with a one-box setup use server='localhost' across all files.  Also a consistent username/password.  Perhaps username='a', password='b' to make it obvious.

Reply:
Updated server to be localhost, username to be 'test' and password to be 'password'